From 496c1740e5475be85b65fdb2b143771f09ed27cb Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Thu, 11 Nov 2021 10:06:31 +0100
Subject: sigsegv: fix build on or1k

Fix the following build failure on or1k with diffutils 3.8
https://git.savannah.gnu.org/cgit/gnulib.git/commit/?id=e54aa6196947ed22ff66bcd714e4fc7bd0c5c3b4
sigsegv.c:979:28: error: 'struct vma_struct' has no member named 'is_near_this'
http://autobuild.buildroot.org/results/5427e52489d29ed4f18b4a70b3c36f0f7bb0ddd9

[Retrieved from:
https://git.savannah.gnu.org/cgit/gnulib.git/commit/?id=496c1740e5475be85b65fdb2b143771f09ed27cb]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 m4/stack-direction.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 70286c8f5..29e398d77 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -45,6 +45,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       nios2 | \
       nds32* | \
       ns32k | \
+      or1k | \
       pdp11 | \
       pj* | \
       powerpc* | rs6000 | \
-- 
cgit v1.2.1

